### ISSUE TITLE
two samples for normal ATAC-Seq

### ISSUE THREAD

**Issue by dwb0211** [GreenleafLab/chromVAR#33]
Is it applicable for two samples to calculate the variability in normal ATAC-Seq but not single cell sequence?

**Comment by AliciaSchep** [comment]
I'm not sure I understand the question -- do you have two samples total?

**Comment by dwb0211** [comment]
Yes, I have one sample for control from normal ATAC-Seq in cultured cells and one for treatment group. chromvar is finished with no error, I want to know whether this result is right or not.
The figure is attache as below:

![chromvar_atac](https://user-images.githubusercontent.com/3456901/41805475-0b691f10-7679-11e8-933b-d2665af0eeef.png)

**Comment by caleblareau** [comment]
I would say that it's technically right in that the variability is a measure that can be applied to two samples. However, you probably almost certainly have some technical confounder there. How deeply did you sequence these two samples (or what's the `colSums` of the counts matrix)? Are the hyper-variable motifs matching at many or a few places?

**Comment by dwb0211** [comment]
@caleblareau  Thanks you so much for your explantation. There are 20M reads per sample and the result match with other methods.

**Comment by ATpoint** [comment]
> I would say that it's technically right in that the variability is a measure that can be applied to two samples. However, you probably almost certainly have some technical confounder there. How deeply did you sequence these two samples (or what's the `colSums` of the counts matrix)? Are the hyper-variable motifs matching at many or a few places?

Could you give an example of such a technical confounding event? Is this based on you experience? We recently used chromVAR for a comparison between two cell types (knockout vs wildtype situation) and the top-variable motifs (in both directions, = more and less accessable motifs)  comprised both the knocked-out factor and TFs that were later supported by independent in silico and wetlab experiments.
